packages/z/zeromq/xmake.lua

```diff
@@ -8,6 +8,8 @@ package("zeromq")
     add_versions("4.3.2", "ebd7b5c830d6428956b67a0454a7f8cbed1de74b3b01e5c33c5378e22740f763")
     add_versions("4.3.4", "c593001a89f5a85dd2ddf564805deb860e02471171b3f204944857336295c3e5")
 
+    add_patches("4.3.4", "https://github.com/zeromq/libzmq/commit/438d5d88392baffa6c2c5e0737d9de19d6686f0d.patch", 
```

Comment:
Will the shasum of the github commit patch be changed in the future?

Reply:
I hope not, there are a few xmake packages that relies on github patches and seem just fine